ruby-training/trainees/david/test_katas/01/calculator.rb

```diff
@@ -0,0 +1,23 @@
+class Calculator
+
+    def self.add(a, b)
+        return nil if b.is_a?(String) or a.is_a?(String)
+        a + b
+    end
+    
+    def self.subtract(a, b)
+        return nil if b.is_a?(String) or a.is_a?(String)
+        a - b
+    end
+    
+    def self.multiply(a, b)
+        return nil if b.is_a?(String) or a.is_a?(String)
```

Comment:
You use this specific line of code a lot, which isn't necessarily bad, but it might be good to refactor it. Maybe turning the conditional argument into a single method like so:

```ruby
def either_operand_is_string?(left_operand, right_operand)
  left_operand.is_a?(String) || right_operand.is_a?(String)
end
```

and then:

```ruby
return nil if either_operand_is_string?(left_operand, right_operand)
```